### ISSUE TITLE
Prevent Editing Phone Number

### ISSUE THREAD

**Issue by zoro238** [hbb20/CountryCodePickerProject#269]
How to Prevent Editing Phone Number. Any readable only
How to make a left-to-right direction always
Programmatically

**Comment by hbb20** [comment]
Developer has the control of Carrier number edittext so it can be frozen by disabling the edittext.

Can you please explain what you meant by  "How to make a left-to-right direction always Programmatically"?

**Comment by zoro238** [comment]
When the language changes, the location of the CCP changes from right  to left or from left to right. I want it to remain fixed from left to right in any language of the phone.

**Comment by zoro238** [comment]
> manifest

ok , but Can I cancel RTL these only?
Because I do not want to cancel RTL the entire application

**Comment by hbb20** [comment]
No. This layout was created according to Android guidelines so it follows
platform's RTL instructions.

**Comment by zoro238** [comment]
OK

**Comment by zoro238** [comment]
OK
